packages/app/src/app/components/WorkspaceSelect/WorkspaceSelect.tsx

```diff
@@ -77,30 +79,16 @@ export const WorkspaceSelect: React.FC<WorkspaceSelectProps> = React.memo(
             <Menu.List
               css={css({
                 width: '100%',
-                marginLeft: 2,
```

Comment:
I think that because of this change the menu is not neatly connected to the workspace item:

![image](https://user-images.githubusercontent.com/587016/117133687-daadd800-ada4-11eb-861f-840b52c64eac.png)

It's a bit too much wide and a bit lower than the selector.

Reply:
I'm not entirely sure what you meant, so among these two versions, which one makes more sense for you?

WIth marginLeft:
![Screenshot 2021-05-05 at 15 24 47](https://user-images.githubusercontent.com/4838076/117156967-18ffc300-adb6-11eb-950b-91386d340b6b.png)

With an extra marginTop:
![Screenshot 2021-05-05 at 15 26 24](https://user-images.githubusercontent.com/4838076/117157238-56645080-adb6-11eb-8484-238439091dfb.png)